src/iqm/qiskit_iqm/iqm_provider.py

```diff
@@ -338,13 +359,17 @@ def __init__(self, url: str, **user_auth_args):  # contains keyword args auth_se
         self.url = url
         self.user_auth_args = user_auth_args
 
-    def get_backend(self, name: Optional[str] = None) -> Union[IQMBackend, IQMFacadeBackend]:
+    def get_backend(
+        self, name: Optional[str] = None, calibration_set_id: Optional[UUID] = None
+    ) -> Union[IQMBackend, IQMFacadeBackend]:
         """An IQMBackend instance associated with this provider.
 
         Args:
             name: optional name of a custom facade backend
+            calibration_set_id: ID of the calibration set used to create the transpilation target of the backend.
+                If None, the server default calibration set will be used.
         """
         client = IQMClient(self.url, client_signature=f'qiskit-iqm {__version__}', **self.user_auth_args)
         if name == 'facade_adonis':
```

Comment:
Maybe this should be the name for the IQM facade backend.

Reply:
I think it's intentionally `facade_adonis`, it's documented in User Guide. I would not change it here.